frontend/src/pages/EditUserPage.test.tsx

```diff
@@ -0,0 +1,99 @@
+import EditUserPage from './EditUserPage';
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import Enzyme from 'enzyme';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { ILookupCode } from 'actions/lookupActions';
+import * as API from 'constants/API';
+import { Provider } from 'react-redux';
+import * as reducerTypes from 'constants/reducerTypes';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+const mockStore = configureMockStore([thunk]);
+
+const lCodes = {
+  lookupCodes: [
+    { name: 'agencyVal', id: '1', isDisabled: false, type: API.AGENCY_CODE_SET_NAME },
+    { name: 'roleVal', id: '1', isDisabled: false, type: API.ROLE_CODE_SET_NAME },
+  ] as ILookupCode[],
+};
+
+const selectedUser = {
+  username: 'test.user',
+  firstName: 'Test',
+  lastName: 'User',
+  email: 'test@user.com',
+  isDisabled: false,
+  emailVerified: false,
+  agencies: [],
+  roles: [],
+  rowVersion: 'AAAAAAAAB9E=',
+};
+
+const store = mockStore({
+  [reducerTypes.GET_USER_DETAIL]: selectedUser,
+  [reducerTypes.LOOKUP_CODE]: lCodes,
+});
+
+const component = mount(
+  <Provider store={store}>
+    <EditUserPage id="41a46ea3-e815-4272-9a17-693ba2409841" />,
```

Comment:
What's the hard-coded guid here for?

Reply:
That would be an unchanged copy and paste, thanks for catching that!